assets/css/app.css

```diff
@@ -106,6 +106,10 @@
   animation: 0.2s ease-out 0s normal forwards 1 fade-out-keys;
 }
 
+.flex-col-reverse {
+  flex-direction: column-reverse;
+}
```

Comment:
@josephjclark isn't this already in tailwind?

Reply:
removing as this doesn't appear to be used anywhere in the app :-)